src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -689,7 +698,93 @@ protected bool AccessorsHaveImplementation
         /// a property with an accessor using the 'field' keyword, or
         /// a property with an initializer.
         /// </summary>
-        internal SynthesizedBackingFieldSymbol BackingField { get; }
+        internal SynthesizedBackingFieldSymbol BackingField => _Safe_MergedAutoPropertyInfo.BackingField;
+
+        private AutoPropertyInfo _Safe_MergedAutoPropertyInfo
```

Comment:
What makes this `Safe`?

Reply:
It should avoid a `NullableReferenceException` in the caller in the primary constructor case.